<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>哈夫曼编码处理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/actions/filefind.png</normaloff>:/actions/filefind.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="mainTextView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="pNew"/>
   <addaction name="pOpen"/>
   <addaction name="pCompressSave"/>
   <addaction name="pUnzip"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="pOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/actions/document-open.png</normaloff>:/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="pCompressSave">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/actions/down.png</normaloff>:/actions/down.png</iconset>
   </property>
   <property name="text">
    <string>压缩保存</string>
   </property>
  </action>
  <action name="pUnzip">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/actions/go-up.png</normaloff>:/actions/go-up.png</iconset>
   </property>
   <property name="text">
    <string>解码保存</string>
   </property>
  </action>
  <action name="pNew">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/actions/document-new.png</normaloff>:/actions/document-new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
